MAINTAINERS.md

```diff
@@ -1,6 +1,139 @@
+| Can participate in promotions of<br> Code Owners and  Maintainers                                                                           |            ✗            |            ✗           |            ✓            |
+| Can suggest Milestones during planning                                                                                                      |            ✓            |            ✓           |            ✓            |
+| Can choose Milestones for specific component                                                                                                |            ✗            |            ✓           |            ✓            |
+| Choose project's Milestones during planning                                                                                                 |            ✗            |            ✗           |            ✓            |
+| Can propose new RFC or<br> participate in review of existing RFC                                                                            |            ✓            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in represented area of responsibility                                                                        |            ✗            |            ✓           |            ✓            |
+| Can request rework of RFCs<br> in any part of the project                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can manage release process of the project                                                                                                   |            ✗            |            ✗           |            ✓            |
+| Can represent the project in public as a Maintainer                                                                                         |            ✗            |            ✗           |            ✓            |
+
+These roles are merit based. Refer to the corresponding section for specific
+requirements and the nomination process.
+
+## <a name="contributor"></a>Contributor
+
+A Contributor can invest their time and resources in several different ways to improve oneDPL
+* Answering questions from community members in Discussions on Issues
+* Providing feedback on RFC pull requests and Discussions
+* Reviewing and/or testing pull requests
+* Contribute code, including bug fixes, new examples of oneDPL use, and new feature implementations
+* Contribute design proposals as RFCs
+
+Responsibilities:
+* Follow the [Code of Conduct](CODE_OF_CONDUCT.md)
+* Submit issues, feature requests, and code in accordance with the [Contribution Guidelines](CONTRIBUTING.md)
+
+Privileges:
+* Code contributions recognized in the oneDPL [credits](CREDITS.txt)
+* Eligible to become a Code Owner
+* Read permissions granted to the repository
+* Can suggest Milestones during planning
```

Comment:
Yes, I expect milestones to continue to be release-oriented.

Reply:
So then the question I have is what does it mean to "Suggest Milestones during planning" if Milestones are just tied to releases?  Perhaps the language we have here around milestones don't make as much sense for oneDPL.

Reply:
This was addressed in my commit.  Rather than suggesting milestones the intent is to allow contributors to suggest PRs and Issues that should be addressed in a milestone.

Reply:
Looks good.